tests/generators/bls/main.py

```diff
@@ -120,42 +145,46 @@ def case02_verify():
                 'output': False,
             }
 
-        # Valid pubkey and signature with the point at infinity
-        assert bls.Verify(Z1_PUBKEY, message, Z2_SIGNATURE)
-        assert milagro_bls.Verify(Z1_PUBKEY, message, Z2_SIGNATURE)
-        yield f'verify_infinity_pubkey_and_infinity_signature', {
-            'input': {
-                'pubkey': encode_hex(Z1_PUBKEY),
-                'message': encode_hex(message),
-                'signature': encode_hex(Z2_SIGNATURE),
-            },
-            'output': True,
-        }
+    # Invalid pubkey and signature with the point at infinity
+    assert not bls.Verify(Z1_PUBKEY, SAMPLE_MESSAGE, Z2_SIGNATURE)
+    # assert not milagro_bls.Verify(Z1_PUBKEY, SAMPLE_MESSAGE, Z2_SIGNATURE)  # TODO: enable it when milagro is ready
+    yield f'verify_infinity_pubkey_and_infinity_signature', {
+        'input': {
+            'pubkey': encode_hex(Z1_PUBKEY),
+            'message': encode_hex(SAMPLE_MESSAGE),
+            'signature': encode_hex(Z2_SIGNATURE),
+        },
+        'output': False,
+    }
 
 
 def case03_aggregate():
     for message in MESSAGES:
         sigs = [bls.Sign(privkey, message) for privkey in PRIVKEYS]
+        aggregate_sig = bls.Aggregate(sigs)
+        assert aggregate_sig == milagro_bls.Aggregate(sigs)
         yield f'aggregate_{encode_hex(message)}', {
             'input': [encode_hex(sig) for sig in sigs],
-            'output': encode_hex(bls.Aggregate(sigs)),
+            'output': encode_hex(aggregate_sig),
         }
 
     # Invalid pubkeys -- len(pubkeys) == 0
-    try:
-        bls.Aggregate([])
-    except Exception:
-        pass
-    else:
-        raise Exception("Should have been INVALID")
-
+    expect_exception(bls.Aggregate, [])
     # No signatures to aggregate. Follow IETF BLS spec, return `None` to represent INVALID.
-    # https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-02#section-2.8
+    # https://tools.ietf.org/html/draft-irtf-cfrg-bls-signature-04#section-2.8
     yield f'aggregate_na_signatures', {
         'input': [],
         'output': None,
     }
 
+    # Valid to aggregate G2 point at infinity
```

Comment:
I've opened an issue to get confirmation from IETF BLS standard maintainers: cfrg/draft-irtf-cfrg-bls-signature#31